core/src/main/resources/error/error-classes.json

```diff
@@ -63,5 +63,86 @@
   "WRITING_JOB_ABORTED" : {
     "message" : [ "Writing job aborted" ],
     "sqlState" : "40000"
+  },
+  "INVALID_INSERT_INTO_CONTEXT" : {
+    "message" : [ "Invalid InsertIntoContext" ],
+    "sqlState" : "10001"
```

Comment:
The SQLSTATEs should reflect those set in the ANSI/ISO standard; see https://github.com/apache/spark/blob/master/core/src/main/resources/error/README.md.

Reply:
I'm not sure.